src/lang/modifyAst/addSweep.ts

```diff
@@ -0,0 +1,399 @@
+import type { Node } from '@rust/kcl-lib/bindings/Node'
+
+import {
+  createArrayExpression,
+  createCallExpressionStdLibKw,
+  createLabeledArg,
+  createLiteral,
+  createLocalName,
+  createVariableDeclaration,
+  findUniqueName,
+} from '@src/lang/create'
+import {
+  getEdgeTagCall,
+  mutateAstWithTagForSketchSegment,
+} from '@src/lang/modifyAst/addEdgeTreatment'
+import {
+  getNodeFromPath,
+  getProfileExpressionsFromSelection,
+  valueOrVariable,
+} from '@src/lang/queryAst'
+import { ARG_INDEX_FIELD, LABELED_ARG_FIELD } from '@src/lang/queryAstConstants'
+import { getNodePathFromSourceRange } from '@src/lang/queryAstNodePathUtils'
+import type {
+  CallExpressionKw,
+  Expr,
+  PathToNode,
+  Program,
+  VariableDeclaration,
+} from '@src/lang/wasm'
+import type {
+  KclCommandValue,
+  KclExpressionWithVariable,
+} from '@src/lib/commandTypes'
+import { KCL_DEFAULT_CONSTANT_PREFIXES } from '@src/lib/constants'
+import type { Selections } from '@src/lib/selections'
+import { err } from '@src/lib/trap'
+
+export function addExtrude({
```

Comment:
`addExtrude` is a good default case here to showcase the pattern:
1. we clone the ast so we can edit it
2. we map the sketches selection into a list of kcl expressions to be passed as unlabelled argument to sweep calls
3. if needed: we insert variable(s) for labeled args
4. if edit, we _assign_ the new function call declaration to the existing node, otherwise just push to the end
5. we return the cloned ast and the path to node

Reply:
You're totally right, added them!